api/provider/aws/deploy_read.go

```diff
@@ -9,6 +9,9 @@ import (
 	"github.com/quintilesims/layer0/common/models"
 )
 
+// Read returns a Deploy based on the provided Deploy ID. The Deploy ID is used to look up
```

Comment:
This one gave me slight pause --- seems like more context around what a 'Deploy' is would be useful if this is going into GoDoc style docs

Reply:
Should we then make a comment in the file `common/models/deploy.go` above the `struct` definition of Deploy to provide this context?

Reply:
:thinking: I don't think so, otherwise we'd be inclined to add comments to all (or most) of the models, which seems a bit heavy-handed at this point. If anyone has thoughts on that, I'm all 👂 s.

Reply:
I think what I'm going to do here is follow what I did for Task Read() and say "Read returns a models.Deploy based on the provided Deploy ID." rather than just "returns a Deploy". That way it is clearer to the reader that I'm returning a Deploy as defined within the `models` package. I'll make some minor adjustments to the explanation as well.